core/trino-main/src/main/java/io/trino/server/security/AbstractBearerAuthenticator.java

```diff
@@ -49,13 +50,12 @@ public Identity authenticate(ContainerRequestContext request, String token)
             throws AuthenticationException
     {
         try {
-            Map<String, Object> claims = convertTokenToClaims(token);
-            Object principalObj = claims.get(principalField);
-            if (principalObj == null) {
+            Optional<Map<String, Object>> claims = convertTokenToClaims(token);
```

Comment:
Why not just return an empty map since the result is the same?

Reply:
👍